feature-libs/product-configurator/common/components/configure-product/configure-product.component.ts

```diff
@@ -30,8 +30,36 @@ export class ConfigureProductComponent {
 
   ownerTypeProduct: CommonConfigurator.OwnerType =
     CommonConfigurator.OwnerType.PRODUCT;
-
+  /**
+   * Prevents page down behaviour when users press space key to select buttons
+   *
+   * @param {KeyboardEvent} event - Keyboard event
+   * @param {Configurator.Group} group - Entered group
```

Comment:
Please, remove this parameter because the method gets only an event parameter

Reply:
Done